parasol_runtime/src/fluent/generic_int.rs

```diff
@@ -714,6 +905,87 @@ where
     }
 }
 
+#[derive(Clone, Serialize, Deserialize)]
+/// Similar to [`GenericInt`] but without the size N generic parameter
+pub struct DynamicGenericInt<T: CiphertextOps, U: Sign> {
+    /// The ciphertexts encrypting this generic integer's bits in least-to-most significant order.
+    pub bits: Vec<Arc<AtomicRefCell<T>>>,
+    _phantom: PhantomData<U>,
+}
+
+impl<T, U> DynamicGenericInt<T, U>
```

Comment:
Idea for another CR, migrate the existing `GenericInt<N>` types be a simple wrapper around this type that passes `N` for `n`. This removes code duplication.